fetchai/ledger/contract.py

```diff
@@ -89,7 +89,7 @@ def create(self, api: ContractsApiLike, owner: Entity, fee: int):
             resource_addresses.extend(ShardMask.state_to_address(address, self) for address in
                                       self._parser.used_globals_to_addresses(self._init, [self._owner]))
         except (UnparsableAddress, UseWildcardShardMask):
-            print("WARNING: Couldn't auto-detect used shards, using wildcard shard mask")
+            logging.warn("Couldn't auto-detect used shards, using wildcard shard mask")
```

Comment:
logging.warn is deprecated, use logging.warning instead.
```suggestion
            logging.warning("Couldn't auto-detect used shards, using wildcard shard mask")
```

Reply:
Done.